### ISSUE TITLE
Question: If the edge on linux would be supported

### ISSUE THREAD

**Issue by TangdizhiH** [keepassxreboot/keepassxc-browser#1332]
Hello, dear developer of Keepassxc,

I'm one of the user of your application and thank you for the excellent application. I just want to ask if you have a plan to support the edge on linux, which has already enter its beta channel for all users. It is the browser in my workflow and it would be extremely convenient if I can connect these two apps. But right now edge seems to be the only main-stream browser that is not supported, although its quiet similar to Google Chrome. The chrome extension doesn't work on edge on linux, it cannot establish connection with keepassXC. So I want to ask about the plan of your developer. Thank you.

**Comment by droidmonkey** [comment]
Not until it's out of beta, use the custom browser option in the browser integration advanced settings. There is another issue tracking edge browser integration.
